<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Transformacia Zameranych Bodov</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="6,2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="lavapolovica">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="labelPredloha">
             <property name="text">
              <string>Body Predlohy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditPredloha">
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="labelMeranie">
             <property name="text">
              <string>Body z Merania</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditMeranie"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelSubor2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Vstup 2
Meranie</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSubor2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSubor2">
           <property name="text">
            <string>Hladat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelSubor1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Vstup 1
Predloha</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSubor1"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSubor1">
           <property name="text">
            <string>Hladat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="labelZachytne">
         <property name="text">
          <string>Zachytne Body ( z Merania ) [ Vyber Riadok : Ctrl + Left Click ]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetZachytne">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonObnovit">
         <property name="text">
          <string>Obnovit Tabulku</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonVsetky">
         <property name="text">
          <string>Vyber Vsetky</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>LSQ</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelChyba">
               <property name="text">
                <string>Chyba vypoctu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditChyba">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string extracomment="MSE - Vazena stredna chyba z euklidovskej vzdialenosti transformovanych sustavy zachytnych bodov merania do sustavy bodov predlohy od prisluchajucich bodov predlohy"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="plainTextEditTransformovane">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>plainTextEditTransformovane</zorder>
          <zorder></zorder>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Posun, Rotacia, Rotacia</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="8" column="0">
            <widget class="QPlainTextEdit" name="plainTextEditTransformovane_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="labelChyba_2">
               <property name="text">
                <string>Chyba vypoctu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditChyba_2">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string extracomment="MSE - Vazena stredna chyba z euklidovskej vzdialenosti transformovanych sustavy zachytnych bodov merania do sustavy bodov predlohy od prisluchajucich bodov predlohy"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QRadioButton" name="radioButton3bod">
               <property name="text">
                <string>3. bod</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit3bod">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QRadioButton" name="radioButton2bod">
               <property name="text">
                <string>2. bod</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit2bod">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QRadioButton" name="radioButton1bod">
               <property name="text">
                <string>1. bod</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit1bod">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonNA">
             <property name="text">
              <string>Ziaden vyber</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTransformovane">
         <property name="text">
          <string>Transformovane Body</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelPresnost">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Presnost - Pocet Desatinnych miest</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxPresnost">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPrepocitat">
         <property name="text">
          <string>Prepocitat</string>
         </property>
         <property name="iconSize">
          <size>
           <width>124</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUkazka">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Ukazka</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUlozit">
         <property name="text">
          <string>Ulozit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
